addons/website_event/tests/test_event_internals.py

```diff
@@ -30,18 +46,10 @@ def test_menu_management_frontend(self):
             'website_menu': True,
             'community_menu': False,
         })
-        self.assertTrue(event.website_menu)
-        self._assert_website_menus(event)
+        self._assert_website_menus(event, ['Introduction', 'Location', 'Register'], menus_out=['Community'])
 
         introduction_menu = event.menu_id.child_id.filtered(lambda menu: menu.name == 'Introduction')
         introduction_menu.unlink()
 
         self.assertTrue(event.website_menu)
```

Comment:
Would also be nice to test that if you flip your new "introduction_menu" field back to True, it re-creates the introduction menu.